### ISSUE TITLE
[Reading] RLHF Can Speak Many Languages: Unlocking Multilingual Preference Optimization for LLMs

### ISSUE THREAD

**Issue by hiroaki-1-1-0** [hiroaki-1-1-0/Reading-Papers#1]
## 0. Info
- Author: John Dang, Arash Ahmadian, Kelly Marchisio, Julia Kreutzer, Ahmet Üstün, Sara Hooker
- Paper: [link](https://aclanthology.org/2024.emnlp-main.729/)

## 1. Overview
- Multilingual reference optimization for LLMs, using multilingual feedback.

## 2. Novelty / Utility
- Proposing scalable method for generating high-quality multilingual feedback data to balance data coverage, without relying on translationese.
- The inclusion of multilingual data leads to improvements in multilingual performance and cross-lingual transfer. This improvement scales both with the number of languages and the total number of examples included in the training data.

## 3. Core Method
- This research constructs a diverse set of general instruction-following multilingual prompts by translating approximately 50K English prompts into 22 other languages, using LLMs.
- After translating prompts, they generate completions for each language by using multiple LLMs of varying multilingual capability.

## 4. Validation
- Comparison of win-rate against SOAT multilingual LLM, judged by LLM.
<img width="511" height="547" alt="Image" src="https://github.com/user-attachments/assets/5041536f-21f1-4a4b-8e29-b3ff3db64220" />
 
- Comparison of win-rate by number of training languages, judged by LLM.
<img width="1028" height="423" alt="Image" src="https://github.com/user-attachments/assets/31d08156-b500-44f0-b119-02a157d53dd5" />

## 5. Limitations / Discussions
- This research used 23 languages, but there are approximately 7,000 languages in the world.
- This research didn't distinguish between dialects within the languages.
- Due to compute constraints, this research couldn't use large models and couldn't tune fine hyperparameters.

## 6. Further Reading
- MAPO: Advancing Multilingual Reasoning through Multilingual Alignment-as-Preference Optimization
- LLMs Beyond English: Scaling the Multilingual Capability of LLMs with Cross-Lingual Feedback
- Cross-lingual Transfer of Reward Models in Multilingual Alignment

## 7. Personal Comments
- This research doesn't use the dataset for comparing outputs from the same model, but rather use the dataset to compare direct translation by Command R+ and indirect translation by Command.
- When languages other than English are mixed into the online learning data of multilingual models, English performance improves. I hypothesize this occurs because through distillation, the model learns from the differences between Command's translation skills and Command R+'s translation skills. To verify this phenomenon, we should conduct experiments using data generated by human or a single model.
- The English △W-L box in Table 13 should be shaded light green.
